### ISSUE TITLE
Projection issue in backgroundmaps

### ISSUE THREAD

**Issue by dbrakenhoff** [ArtesiaWater/nlmod#240]
When adding a backgroundmap the result is shifted slightly relative to data that was already in the EPSG:28992 coordinate reference system. By supplying the correct CRS proj string this issue is fixed. nlmod uses this new modified CRS string by default. 

Old:
![image](https://github.com/ArtesiaWater/nlmod/assets/6552225/a51104c4-e694-4b2c-9034-fa494be83658)

New:
![image](https://github.com/ArtesiaWater/nlmod/assets/6552225/d65b011d-32f4-4c60-916d-330332446392)

Fix from this issue: https://gis.stackexchange.com/questions/348339/using-crs-epsg3857-but-misalignment-between-stamen-background-and-coordinates-o

**Comment by dbrakenhoff** [comment]
Closed by #241

**Comment by dbrakenhoff** [comment]
For future reference:
```
import nlmod

extent = [110_000, 120_000, 485_000, 495_000]
ds = nlmod.get_ds(extent, crs=28992)
gdf_opp_water = nlmod.read.rws.get_gdf_surface_water(ds)

f, ax = nlmod.plot.get_map(ds.extent)
gdf_opp_water.plot(hatch="///", facecolor="none", ax=ax)
nlmod.plot.add_background_map(ax=ax, crs=28992)
```
